src/cbc_sdk/helpers.py

```diff
@@ -92,7 +92,7 @@ def get_cb_cloud_object(args):
     return cb
 
 
-def get_object_by_name_or_id(cb, cls, name_field="name", id=None, name=None, force_init=True):
+def get_object_by_name_or_id(cb, cls, name_field="name", id=None, name='', force_init=True):
```

Comment:
If name is not provided, then it fails to format it - attempted_to_find = "name {0:s}".format(name) and then fails with the error that  attempted_to_find is used before assignment.